internal/pkg/app/handlers_v1.go

```diff
@@ -315,46 +293,51 @@ func (a *APIManager) setTenantConfigHandler(w http.ResponseWriter, r *http.Reque
 		resp.Respond(ctx, w)
 		return
 	}
-
 	a.quotaManager.PublishQuota(ctx, *tid)
-
 	resp := ItemResponse(tenantConfig)
 	resp.Respond(ctx, w)
 }
 
 func (a *APIManager) deleteTenantConfigHandler(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 	vars := mux.Vars(r)
-
 	tid, apiErr := getTenant(ctx, vars)
 	if apiErr != nil {
 		log.Ctx(ctx).Error().Str("op", "deleteTenantConfigHandler").Str("error", apiErr.Error()).Msg("orgId or appId empty")
 		resp := ErrorResponse(apiErr)
 		resp.Respond(ctx, w)
 		return
 	}
-
 	err := a.tenantStorer.DeleteConfig(ctx, *tid)
 	if err != nil {
 		log.Ctx(ctx).Error().Str("op", "deleteTenantConfigHandler").Str("error", err.Error()).Msg("error deleting tenant config")
 		resp := ErrorResponse(convertToApiError(err))
 		resp.Respond(ctx, w)
 		return
 	}
-
 	resp := ItemResponse(tid)
 	resp.Respond(ctx, w)
 }
 
 func convertToApiError(err error) ApiError {
 	var tenantNotFound *tenant.TenantNotFoundError
-	var badConfig *tenant.BadConfigError
+	var badTenantConfig *tenant.BadConfigError
+	var badRouteConfig *tablemgr.BadConfigError
+	var routeValidationError *tablemgr.RouteValidationError
+	var routeRegistrationError *tablemgr.RouteRegistrationError
+	var routeNotFound *route.RouteNotFoundError
```

Comment:
Mmm ... I wonder for consistency that we should have a tablemgr.RouteNotFoundError that encapsulates the route.RouteNotFoundError. I will let you decide :)